src/tactic/mk_iff_of_inductive_prop.lean

```diff
@@ -76,8 +87,9 @@ meta def constr_to_prop (univs : list level) (g : list expr) (idxs : list expr)
   end,
   return ((bs, n), r)
 
-private meta def to_cases (s : list $ list (option expr) × (expr ⊕ ℕ)) : tactic unit := do
-  h ← intro1,
+@[nolint doc_blame] -- TODO: document
+meta def to_cases (s : list $ list (option expr) × (expr ⊕ ℕ)) : tactic unit :=
```

Comment:
Any docstring suggestions?

Reply:
These will take some studying to figure out, and I don't think we'll get the original author to chime in. I don't mind leaving it `nolint`ed for now, and doing that studying if and when it's necessary.

Reply:
OK, fair enough. I'd be OK merging this for now then.